### ISSUE TITLE
Selenium tests failing

### ISSUE THREAD

**Issue by SergioBertolinSG** [owncloud/core#27749]
Hello @individual-it,  we have some problems with your tests in master.

Check this one for example
https://saucelabs.com/beta/tests/e6e93c363cca4fdfb7fa324c350da51a/commands#85

Maybe postgresql is returning a timeout when installing?

I am also questioning if this variable should be set SRV_HOST_URL.

owncloud:8888/index.php/login this is returning 404, so the installation didn't went well or the path is wrong.

When checking the current path I got /home/travis/build/owncloud/core, maybe
SRV_HOST_URL should be owncloud/core ?

I think it would be better to run them in mysql, but when changing it there is an error:

```
{"reqId":"uCtUPFp9K2W8HaMzfclN","level":3,"time":"2017-04-26T14:51:09+00:00","remoteAddr":"","user":"--","app":"mysql.setup","method":"--","url":"--","message":"Specific user creation failed: An exception occurred while executing 'SELECT user FROM mysql.user WHERE user=?' with params [\"oc_admin\"]:\n\nSQLSTATE[42000]: Syntax error or access violation: 1142 SELECT command denied to user 'oc_autotest'@'localhost' for table 'user'"}
```

**Comment by individual-it** [comment]
php PHP webserver is not starting. After displaying the error messages of it I get
`Failed to listen on owncloud:8888 (reason: Address already in use)`

**Comment by SergioBertolinSG** [comment]
In the integration tests we add a jenkins variable to avoid collisions: 
PORT=$((8080 + $EXECUTOR_NUMBER))

And after the tests we kill the server process.

Maybe travis requires this also?

**Comment by DeepDiver1975** [comment]
Travis is running in its own vm/docker. No port collision possible

**Comment by individual-it** [comment]
changing the port to 8889 did fix the problem. I guess some other script starts also some server on 8888

**Comment by individual-it** [comment]
sauce connect seems to listen on port 8888
`tcp        0      0 127.0.0.1:8888          0.0.0.0:*               LISTEN      3895/sc`

**Comment by individual-it** [comment]
fixed by #27751
